ios/PresentationLayer/UIToolkit/Sources/UIToolkit/BaseViews/SwiftUI/ToastView/ToastView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Created by Tomas Brand on 02.11.2022
+//  Copyright © 2022 Matee. All rights reserved.
+//
+
+import SwiftUI
+
+struct ToastView: View {
+    
+    private let data: ToastData
+    
+    init(_ data: ToastData) {
+        self.data = data
+    }
+    
+    var body: some View {
+        VStack(alignment: .center) {
+            HStack(alignment: .top) {
+                data.style.image
+                    .foregroundColor(data.style.color)
+                
+                Text(data.title)
+                    .foregroundColor(AppTheme.Colors.text)
+                    .font(.system(size: 14, weight: .semibold))
+            }
+            .padding()
+        }
+        .background(
+            Capsule()
+                .fill(.white)
+                .clipped()
+        )
+        .frame(minWidth: 0, maxWidth: .infinity)
+        .shadow(color: Color.black.opacity(0.25), radius: 4, x: 0, y: 1)
+        .padding(.horizontal, 16)
+    }
```

Comment:
Maybe I found what's causing the UI glitch where label just appears instead of sliding together with that bubble. When the SwiftUI view changes, it do not changes the whole view, but only it's subparts, that needs to be changed.

In this case, there is a `VStack` as parent and `HStack` as child. The bubble is a `.background` modifier applied to parent and label as `Text` component in a child. The child have all data used by the `ToastView`, so in some cases when the data changes, the view can refresh only it's child because parent do not have anything that needs to be changed, including the `.background` modifier (bubble).

So I suggest to apply those modifiers that creates a bubble, to a same component that stores the data for a label. And that should link the label together with the bubble background. Maybe deleting the `VStack` would be enough in this case.
```
var body: some View {
    HStack(alignment: .top) {
        data.style.image
            .foregroundColor(data.style.color)
        
        Text(data.title)
            .foregroundColor(AppTheme.Colors.text)
            .font(.system(size: 14, weight: .semibold))
    }
    .padding()
    .background(
        Capsule()
            .fill(.white)
            .clipped()
    )
    .frame(minWidth: 0, maxWidth: .infinity)
    .shadow(color: Color.black.opacity(0.25), radius: 4, x: 0, y: 1)
    .padding(.horizontal, 16)
}
```
It's just a suggestion I haven't test it out :)